classes/PodsMeta.php

```diff
@@ -234,6 +236,7 @@ public function core() {
 					if ( ! pods_tableless() ) {
 						add_filter( 'add_post_metadata', array( $this, 'add_post_meta' ), 10, 5 );
 						add_filter( 'update_post_metadata', array( $this, 'update_post_meta' ), 10, 5 );
+						add_filter( 'update_post_metadata_by_id', array( $this, 'update_post_meta_by_id' ), 10, 4 );
```

Comment:
:no_entry_sign: Short array syntax must be used to define arrays

<sub>hash: 770ede8cdb14e9928b91a738b78da094</sub>